pkg/syncer/syncertest/fake/storage.go

```diff
@@ -304,6 +304,21 @@ func (ms *MemoryStorage) listObjects(gk schema.GroupKind) []*unstructured.Unstru
 	return result
 }
 
+func lookupGVK(obj client.Object, scheme *runtime.Scheme) (schema.GroupVersionKind, error) {
+	switch obj.(type) {
+	case *metav1.PartialObjectMetadata:
```

Comment:
wanna add this case to kinds.Lookup?

Reply:
Should be fine. But you might want to do it in a different PR and let the tests run.